### PR TITLE
{math,numlib}[intel/2020b] PEXSI v2.0.0, SuperLU_DIST v7.2.0

### PR DESCRIPTION
**Pull request by Dankomaister**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Dankomaister** [comment]
Fixed `W191 indentation contains tabs` and `E221 multiple spaces before operator`

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/p/PEXSI/PEXSI-2.0.0-intel-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    'filename': 'PEXSI-%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://bitbucket.org/berkeleylab/pexsi',
+        'repo_name': 'PEXSI',
+        'commit': '966f9f9',
+        'recursive': True,
+        'keep_git_dir': True,
+    },
+}]
+checksums = ['eecb5ad065095e480ccb948ae6774bbdd373155648eb83800591b7540bde315d']
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/p/PEXSI/PEXSI-2.0.0-intel-2020b.eb:24]: wrong checksum?

**Comment by boegel** [comment]
closing this since `intel/2020b` is deprecated, see https://docs.easybuild.io/policies/toolchains

**Pull request closed without merging**